cmd/actions/actionscmd.go

```diff
@@ -20,10 +21,11 @@ import (
 )
 
 const (
-	flagGRPC   = "grpc"
-	flagRPC    = "rpc"
-	flagSecure = "secure"
-	flagPort   = "port"
+	flagGRPC     = "grpc"
+	flagRPC      = "rpc"
+	flagSecure   = "secure"
+	flagPort     = "port"
+	flagPortProm = "promport"
```

Comment:
I would rename this flag to `prometheus-port` and maybe add another one that is `enable-prometheus` so that the user can turn it on/off easily